sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/command_component.py

```diff
@@ -133,6 +139,22 @@ def __init__(
                 error_category=ErrorCategory.USER_ERROR,
             )
         self.instance_count = instance_count
+        self.additional_includes = additional_includes
```

Comment:
Not sure whether `additional_includes` should be handled like other attributes. Local path doesn't make sense in remote component.

Reply:
need confirm with long what's "additional_includes " like in remote component, maybe no this field in remote component

Reply:
have create a command component with additional include, seems there's no the field and backend won't return the field for remote component 

https://ml.azure.com/module/test_476709299326/name/test_476709299326?wsid=/subscriptions/96aede12-2f73-41cb-b983-6d11a904839b/resourceGroups/chenyin-test-eastus/providers/Microsoft.MachineLearningServices/workspaces/sdk_vnext_cli&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

Reply:
confirmed with Long, current implementation is okay.
NIT: default value of additional_includes should be an empty list instead of null in v2 component, as user should be able to append the list directly via SDK

Reply:
Thanks, updated.

Reply:
Thanks, updated